packages/core/bootstrap/src/lib/metrics/util.ts

```diff
@@ -39,14 +39,20 @@ export const getFeedId = (input: AdapterRequest): string => {
     base: ['base', 'from', 'coin', 'symbol', 'asset'],
     quote: ['quote', 'to', 'convert'],
   }
-  try {
+
+  // check if string is within array
+  function includesCheck(param: string) {
+    return Object.keys(input.data).includes(param)
+  }
+
+  // run through validdator if input.data object has keys that match potential base and quote parameters
+  if (commonFeedParams.base.some(includesCheck) && commonFeedParams.quote.some(includesCheck)) {
     const validator = new Validator(input, commonFeedParams)
     if (validator.error) {
       logger.debug('Unable to validate feed name')
       return JSON.stringify(input)
     }
```

Comment:
Is this line still necessary if the parameters are checked to be present in line 49? (will it ever error?)

Reply:
I think it won't. Actually, `validator.error` is taking care of the error, so we can remove it 👍